src/Symfony/Component/Notifier/Bridge/SmsBiuras/Tests/SmsBiurasTransportFactoryTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\SmsBiuras\Tests;
+
+use Symfony\Component\Notifier\Bridge\SmsBiuras\SmsBiurasTransportFactory;
+use Symfony\Component\Notifier\Test\TransportFactoryTestCase;
+use Symfony\Component\Notifier\Transport\TransportFactoryInterface;
+
+final class SmsBiurasTransportFactoryTest extends TransportFactoryTestCase
+{
+    /**
+     * @return SmsBiurasTransportFactory
+     */
+    public function createFactory(): TransportFactoryInterface
+    {
+        return new SmsBiurasTransportFactory();
+    }
+
+    public function createProvider(): iterable
+    {
+        yield [
+            'smsbiuras://host.test?from=0611223344&test=1',
+            'smsbiuras://uid:token@host.test?from=0611223344&test=1',
+        ];
+    }
+
+    public function supportsProvider(): iterable
+    {
+        yield [true, 'smsbiuras://uid:token@default?from=0611223344'];
+        yield [false, 'somethingElse://uid:token@default?from=0611223344'];
+    }
+
+    public function missingRequiredOptionProvider(): iterable
+    {
+        yield 'missing option: from' => ['smsbiuras://uid:token@default'];
+    }
+
+    public function unsupportedSchemeProvider(): iterable
+    {
+        yield ['somethingElse://uid:token@default?from=0611223344'];
+        yield ['somethingElse://uid:token@default']; // missing "from" option
+        yield ['somethingElse://uid:token@default']; // missing "test" option
```

Comment:
This line can be removed, test option should be optional

Reply:
Removed 51 line